src/components/organisms/pupil/OakQuizOrderQuestion/OakQuizOrder.tsx

```diff
@@ -0,0 +1,225 @@
+import React, {
+  FC,
+  createContext,
+  memo,
+  useContext,
+  useEffect,
+  useState,
+} from "react";
+import {
+  DndContext,
+  closestCenter,
+  KeyboardSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent,
+  DragOverlay,
+  DragStartEvent,
+  UniqueIdentifier,
+  Announcements,
+  DndContextProps,
+  MouseSensor,
+  TouchSensor,
+} from "@dnd-kit/core";
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  useSortable,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { createPortal } from "react-dom";
+
+import { OakDraggable } from "@/components/molecules/OakDraggable";
+import { OakDroppable } from "@/components/molecules/OakDroppable";
+import { OakFlex } from "@/components/atoms";
+import { OakDragAndDropInstructions } from "@/components/molecules/OakDragAndDropInstructions";
+
+type OakQuizOrderItem = {
+  id: string;
+  label: string;
+};
+
+export type OakQuizOrderProps = {
+  initialItems: OakQuizOrderItem[];
+  onChange?: (items: OakQuizOrderItem[]) => void;
+};
+
+const ConnectedDraggable = memo(({ id, label }: OakQuizOrderItem) => {
+  const {
+    attributes,
+    listeners,
+    setNodeRef,
+    transform,
+    transition,
+    active,
+    over,
+  } = useSortable({ id });
+  const isGhostItem = active?.id === id;
+  const isGhostSlot = over?.id === id;
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+
+  return (
+    <OakDroppable isOver={isGhostSlot}>
+      <OakDraggable
+        ref={setNodeRef}
+        style={style}
+        isDisabled={isGhostItem}
+        {...attributes}
+        {...listeners}
+        aria-describedby={undefined}
+        aria-roledescription="sortable item"
+        aria-pressed={undefined}
+        aria-selected={!!attributes["aria-pressed"]}
+        role="option"
+      >
+        {label}
+      </OakDraggable>
+    </OakDroppable>
+  );
+});
+
+/**
+ * Facilitates DI for the DndContext
+ */
+export const injectDndContext = createContext<FC<DndContextProps>>(DndContext);
+
+/**
+ * A sortable list of items with drag and drop functionality. Items can be dragged over named slots to re-arrange them
+ *
+ * Keyboard navigation is supported with the `tab`, `space` and `arrow` keys
+ */
+export const OakQuizOrder = ({ initialItems, onChange }: OakQuizOrderProps) => {
+  const [items, setItems] = useState<OakQuizOrderItem[]>(initialItems);
+  const [activeId, setActiveId] = useState<string | null>(null);
+  const activeItem = items.find((item) => item.id === activeId);
+  const [prefersReducedMotion, setPrefersReducedMotion] = useState(false);
+  const sensors = useSensors(
+    useSensor(MouseSensor),
+    useSensor(TouchSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates,
+      scrollBehavior: prefersReducedMotion ? "instant" : "smooth",
```

Comment:
A little QOL improvement that matches the intent of `scrolling-behaviour: auto` in the reset styles.